src/features/wallet/TransactionHistoryModal.tsx

```diff
@@ -14,6 +14,16 @@ import clockYellowIcon from '../../common/assets/icons/ico-clock-yellow.svg'
 import crossIcon from '../../common/assets/icons/ico-cross.svg'
 import addressbookIcon from '../../common/assets/icons/ico-addressbook.svg'
 import user2Icon from '../../common/assets/icons/ico-user2.svg'
+import { useAppSelector } from 'redux/hooks'
+import { RootState } from 'redux/store'
+import { TransactionRPC } from 'api/pastel-rpc'
+import {
+  ITransactionRow,
+  ITTransactionResult,
+  TransactionType,
+} from 'types/rpc'
+import dayjs from 'dayjs'
+import Utils from '../../legacy/utils/utils'
```

Comment:
can we move the needed code to new utils instead of import from legacy?

https://github.com/pastelnetwork/pastel-electron-wallet/tree/master/src/common/utils